test/helpers_spec.rb

```diff
@@ -166,14 +166,15 @@
   describe ".write_config" do
     it "writes the current configuration" do
       url = "https://example.com"
+      config = {
+        url:      url,
+        insecure: false
+      }
       expect(Registration::UrlHelpers).to receive(:registration_url) \
         .and_return(url)
       expect(Registration::Helpers).to receive(:insecure_registration) \
         .and_return(false)
-      expect(SUSE::Connect::YaST).to receive(:write_config).with(
-        url:      url,
-        insecure: false
-      )
```

Comment:
well, I am not sure if old form is wrong as it is one of ruby feature to auto convert hash if it is only argument. But maybe lost with ruby3?

Reply:
Since Ruby 3 hashes and keyword arguments are distinct: https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/